app/assets/javascripts/ui_models/app_state/account_menu_state.ts

```diff
@@ -1,30 +1,60 @@
 import { action, computed, makeObservable, observable, runInAction } from 'mobx';
-import { ContentType } from '@node_modules/@standardnotes/snjs';
+import { ApplicationEvent, ContentType } from '@standardnotes/snjs';
 import { WebApplication } from '@/ui_models/application';
-import { SNItem } from '@node_modules/@standardnotes/snjs/dist/@types/models/core/item';
+import { SNItem } from '@standardnotes/snjs/dist/@types/models/core/item';
 
 export class AccountMenuState {
   show = false;
   signingOut = false;
+  server: string | undefined = undefined;
   notesAndTags: SNItem[] = [];
+  isEncryptionEnabled = false;
+  encryptionStatusString = '';
+  isBackupEncrypted = false;
+  showLogin = false;
+  showRegister = false;
 
   constructor(
     private application: WebApplication,
-    appEventListeners: (() => void)[]
+    private appEventListeners: (() => void)[]
   ) {
     makeObservable(this, {
       show: observable,
       signingOut: observable,
+      server: observable,
       notesAndTags: observable,
+      isEncryptionEnabled: observable,
+      encryptionStatusString: observable,
+      isBackupEncrypted: observable,
+      showLogin: observable,
+      showRegister: observable,
 
       setShow: action,
       toggleShow: action,
       setSigningOut: action,
+      setIsEncryptionEnabled: action,
+      setEncryptionStatusString: action,
+      setIsBackupEncrypted: action,
 
       notesAndTagsCount: computed
     });
 
-    appEventListeners.push(
+    this.addAppEventObserver();
+    this.streamNotesAndTags();
+  }
+
+  addAppEventObserver = (): void => {
```

Comment:
Maybe could be called addAppLaunchedEventObserver

Reply:
Done :)